### DISCUSSION TITLE
Any way to query PMTiles at a point?

### DISCUSSION THREAD

**Discussion by stenuto** [protomaps/PMTiles#168] [General]
What would be the best way to query PMTiles at a specific `lat` / `lon`? Similar to Mapbox's Tilequery API: https://docs.mapbox.com/api/maps/tilequery/

**Comment by bdon** [comment]
Convert the `lat`/`lon` to ZXY and then query for one or more tiles. The solution depends on:

* what language and runtime you are using
* choice of zoom level for your use case
* whether you should query for 1 tile or many nearby tiles to avoid edge effects
